Address asciidoctor-dita-vale issues in Administering Satellite (part 2) (#4527)
theforeman/foreman-documentation: guides/common/modules/proc_monitoring-services.adoc

[id="Monitoring_Services_{context}"]
= Monitoring services

[role="_abstract"]

> You can monitor Smart Proxy services to view their status and configuration details.

Reviewer: ```suggestion
You can view the status and configuration details of {SmartProxy} services in {ProjectWebUI}.
```

Your call.
